cves/CVE-2015-0220.yml

```diff
@@ -268,7 +316,12 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    I believe the mistake was a coding mistake that lead to this vulenrability.
```

Comment:
*vulnerability

Reply:
Perhaps replace coding mistake with coding mishaps so you don't have the word mistake repeated too often.